packages/blade/src/components/PreviewWindow/_decision/decisions.md

```diff
@@ -0,0 +1,144 @@
+# Preview Window Decisions
+
+A **Preview Window** is a component used to display a visual preview of an image or any other UI component.
+
+<img src="./preview.png" width="380" alt="preview-thumbnail" />
+<img src="./breakdown.svg" width="380" alt="breakdown" />
+
+---
+
+- [Design](#design)
+- [Preview Window Component](#preview-window-component)
+  - [Preview Window API](#preview-window-api)
+  - [Alternative API](#alternative-api)
+- [Open Questions](#open-questions)
+
+---
+
+## Design
+
+[Figma Link](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=103462-52670&p=f&t=qC2NU4R56lEGsGFB-0)
+
+---
+
+## Preview Window Component
+
+The `PreviewWindow` component is the primary component used to render a live preview of content with optional interactivity and controls.
+
+---
+
+## Preview Window API
+
+| Prop             | Type                                                   | Default | Required | Description                                                                 |
+|------------------|--------------------------------------------------------|---------|----------|-----------------------------------------------------------------------------|
+| `children`       | `React.Element`                                        | —       | Yes      | The component to be rendered inside the preview window.                     |
+| `onFullScreen`   | `() => void`                                           | —       | No       | Callback invoked when the fullscreen button is clicked.                     |
+| `isDragAndZoomEnabled`  | `boolean`                                              | —       | No       | Whether Drag and Zoom functionality is enabled.                                      |
+| `onZoomChange`   | `(newZoom: number) => void`                            | —       | No       | Callback invoked when the zoom level changes.                               |
```

Comment:
Let's use `({ zoom }) => void` for the callback signature, more scalable in future if we want to expose more things from params.